src/cmd-compress

```diff
@@ -44,13 +44,13 @@ imgs_to_compress = []
 for img_format in buildmeta['images']:
     img = buildmeta['images'][img_format]
     file = img['path']
-    filepath = f'builds/{build}/{file}'
+    filepath = os.path.join('builds', build, file)
     if not file.endswith('.gz'):
-        tmpfile = f'{tmpdir}/{file}.gz'
+        tmpfile = os.path.join(tmpdir, (file + '.gz'))
         # SHA256 for uncompressed image was already calculated during 'build'
         img['uncompressed-sha256'] = img['sha256']
         with open(tmpfile, 'wb') as f:
-            run_verbose(['gzip', '-c', f'builds/{build}/{file}'], stdout=f)
+            run_verbose(['gzip', '-c', filepath], stdout=f)
```

Comment:
looks like we should have been using `filepath` here all along?

Reply:
It wasn't wrong if that's what you mean. But it was indeed repeating the same string again. :)

Reply:
Yeah, I made some minor DRY improvements where I could